app/components/modals/tax-info-modal.js

```diff
@@ -21,16 +21,18 @@ export default ModalBase.extend(FormMixin, {
       delay  : false,
       on     : 'blur',
       fields : {
-        tax_name: {
-          rules: [
+        taxName: {
+          identifier : 'tax_name',
+          rules      : [
             {
               type   : 'empty',
               prompt : this.i18n.t('Please give a name')
             }
           ]
         },
-        tax_rate: {
-          rules: [
+        taxRate: {
+          identifier : 'tax_rate',
+          rules      : [
             {
               type   : 'empty',
               prompt : this.i18n.t('Please tell us your tax rate (in %)')
```

Comment:
Please remove the `number` rule for `taxRate`, it wont be required as the input is already a number.